### ISSUE TITLE
Publish On open-vsx.org

### ISSUE THREAD

**Issue by evanoc3** [firefox-devtools/vscode-firefox-debug#214]
I recently switched to using VSCodium, and found that I couldn't get the firefox debugger extension anymore. According to their [documentation](https://github.com/VSCodium/vscodium/blob/master/DOCS.md#extensions-marketplace), it uses a different store: https://open-vsx.org/

Can we get this extension be published there as well?

**Comment by hbenl** [comment]
Sounds good to me.
@digitarald What do you think? I don't see any reason why we shouldn't do this. I just tried it with some of my other extensions and the process is really simple. Giving multiple people the rights to publish is also possible (and also seems to be very simple to manage).

**Comment by digitarald** [comment]
@hbenl agreed, we have no reason to keep this extension limited to a walled garden.

**Comment by t-fritsch** [comment]
hi,

This extension has been published on open-vsx, thanks for that @hbenl, but it is now many versions behind the one on MS marketplace (2 years late) : 

- [MS marketplace](https://marketplace.visualstudio.com/items?itemName=firefox-devtools.vscode-firefox-debug) : 2.9.8 (last updated 21/07/2022)
- [open-vsx](https://open-vsx.org/extension/firefox-devtools/vscode-firefox-debug) : 2.9.1 (last updated 21/08/2020)

This gap is a problem for VSCodium users, there are some reports of people struggling with firefox debugging related to this problem (https://github.com/EclipseFdn/open-vsx.org/issues/966 or https://github.com/firefox-devtools/vscode-firefox-debug/issues/267) (wich I just did all day long before finding this out). 

I'm not used to publishing extensions on open-vsx but it looks like there are [github actions](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions#github-action) to automate deployment. 

It would be so great to have this extension up to date on open-vsx (vscodium is gaining attention because of this vscode issue : https://github.com/iocave/customize-ui/issues/156 and the solution provided by vscodium : https://github.com/VSCodium/vscodium/issues/1390#issuecomment-1407735477)

Thanks !

PS: I didn't open a new issue since this one isn't closed but if you prefer I can open a new one

**Comment by lehni** [comment]
I am using the description here to use the VSCode Marketplace in VScodium. It's suprisingly easy to set it up:

**Comment by t-fritsch** [comment]
Yes using the extension hosted on the microsoft marketplace is a workaround : using VSIX manual download, or switching to the microsoft marketplace with the steps you pointed to.

But the whole point of using VSCodium is escaping Microsoft services and tracking, hence the alternative marketplace

There are a lot of extensions that now publish on the two marketplaces, I think ff debugger should too and keep the two up to date
